### DISCUSSION TITLE
I need help with FaceSwap it goes on pause

### DISCUSSION THREAD

**Discussion by JonathanZzzAi** [lllyasviel/Fooocus#2886] [Q&A] [answered]
Hello, I need help with the Faceswap tool, I can't use it, after a few moments when I click on generate I get an error in the interface and the process pauses in the console.
![Captura de pantalla 2024-05-08 174525](https://github.com/lllyasviel/Fooocus/assets/152336947/3dc3a2d2-cc2e-45b5-85dc-1fe5f417ead1)
![Captura de pantalla 2024-05-08 174450](https://github.com/lllyasviel/Fooocus/assets/152336947/1a561ea4-256b-40a0-a320-3488f0cba943)

**Comment by mashb1t** [comment]
please follow the instructions in the [troubleshooting guide](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md) and check your system resource usage. If you provide your system specs and the full terminal log of Fooocus from startup to crash we'll be able to guide you better how to fix it.

**Reply by JonathanZzzAi** [reply]
my specs are: 8gb RAM, Rtx 3050 Laptop, 4gb VRAM


full terminal log:
C:\Users\plastik\Desktop\focus\Fooocus_win64_2-1-831>.\python_embeded\python.exe -s Fooocus\entry_with_update.py
Already up-to-date
Update succeeded.
[System ARGV] ['Fooocus\\entry_with_update.py']
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)]
Fooocus version: 2.3.1
[Cleanup] Attempting to delete content of temp dir C:\Users\plastik\AppData\Local\Temp\fooocus
[Cleanup] Cleanup successful
Total VRAM 4096 MB, total RAM 7896 MB
Trying to enable lowvram mode because your GPU seems to have 4GB or less. If you don't want this use: --always-normal-vram
Set vram state to: LOW_VRAM
Always offload VRAM
Device: cuda:0 NVIDIA GeForce RTX 3050 Laptop GPU : native
VAE dtype: torch.bfloat16
Using pytorch cross attention
Refiner unloaded.
Running on local URL:  http://127.0.0.1:7865
IMPORTANT: You are using gradio version 3.41.2, however version 4.29.0 is available, please upgrade.
--------

To create a public link, set `share=True` in `launch()`.
model_type EPS
UNet ADM Dimension 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_l.text_projection', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids'}
Base model loaded: C:\Users\plastik\Desktop\focus\Fooocus_win64_2-1-831\Fooocus\models\checkpoints\juggernautXL_v8Rundiffusion.safetensors
Request to load LoRAs [['sd_xl_offset_example-lora_1.0.safetensors', 0.1], ['None', 1.0], ['None', 1.0], ['None', 1.0], ['None', 1.0]] for model [C:\Users\plastik\Desktop\focus\Fooocus_win64_2-1-831\Fooocus\models\checkpoints\juggernautXL_v8Rundiffusion.safetensors].
Loaded LoRA [C:\Users\plastik\Desktop\focus\Fooocus_win64_2-1-831\Fooocus\models\loras\sd_xl_offset_example-lora_1.0.safetensors] for UNet [C:\Users\plastik\Desktop\focus\Fooocus_win64_2-1-831\Fooocus\models\checkpoints\juggernautXL_v8Rundiffusion.safetensors] with 788 keys at weight 0.1.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cpu, use_fp16 = False.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
Started worker with PID 25072
App started successful. Use the app with http://127.0.0.1:7865/ or 127.0.0.1:7865
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ControlNet Softness = 0.25
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 7539592551635895971
[Fooocus] Downloading control models ...
[Fooocus] Loading control models ...
extra clip vision: ['vision_model.embeddings.position_ids']
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 15
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] girl in a park, gorgeous, amazing, elegant, intricate, highly detailed, focus, intense, new, designed, shiny, colorful, inspired, epic, beautiful, cinematic, artistic, fine detail, clear, neat, dynamic composition, light, colors, perfect, expressive, advanced, full color, complex, cool, awesome, attractive, creative, hopeful, marvelous
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] girl in a park, cinematic, dramatic atmosphere, detailed, intricate, elegant, highly detail, sharp focus, beautiful, inspired, rich charming, pretty, draped, artistic, professional, surreal, expressive, cool, iconic, best, perfect, pleasant, cute, futuristic, romantic, peaceful, new, friendly, colorful, color, light, saturated, amazing, symmetry
[Fooocus] Encoding positive #1 ...
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Fooocus] Image processing ...
Detected 1 faces
Requested to load CLIPVisionModelWithProjection
Loading 1 new model
Requested to load Resampler
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 0.50 seconds
Requested to load To_KV
Loading 1 new model
[Fooocus Model Management] Moving model(s) has taken 2.24 seconds
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (896, 1152)
Preparation time: 40.29 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model

C:\Users\plastik\Desktop\focus\Fooocus_win64_2-1-831>pause
Presione una tecla para continuar . . .

**Reply by mashb1t** [reply]
@JonathanZzzAi did you follow the instructions in the troubleshooting guide and can you confirm that swap is set up correctly and in sufficient amount (40GB)?
https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md#connection-errored-out

**Reply by JonathanZzzAi** [reply]
Yes, I already followed the instructions, the system swap was configured correctly and I have 40gb free

**Reply by mashb1t** [reply]
Is the swap on the same drive Fooocus is on (C drive)? It is historically proven that this can be an issue. You can manually check and set swap to a specific drive.

**Reply by JonathanZzzAi** [reply]
I only have one drive (C drive) and I did the swap on that
![Captura de pantalla 2024-05-09 154629](https://github.com/lllyasviel/Fooocus/assets/152336947/e011552b-7ee2-4953-838e-40c710d443d9)
![Captura de pantalla 2024-05-09 154714](https://github.com/lllyasviel/Fooocus/assets/152336947/e90e8e8f-a03a-4554-bf34-d7e46fba592c)

**Reply by mashb1t** [reply, marked as answer]
please test if unchecking automated page size managemebt and manual size of 40GB helps and keep an eye on the task manager disk, ram and GPU VRAM usage

**Reply by JonathanZzzAi** [reply]
Thank you very much, I unchecked the automated page size management, manually placed the 40GB and uninstalled it and reinstalled fooocus and faceswap and it worked
